packages/blocks/src/lib/navigation-map/lib/create-obstacle.ts

```diff
@@ -10,5 +11,7 @@ export const createObstacle = (
     name,
     location: { lng: lngLat.lng, lat: lngLat.lat },
     geometries: [createGeometry(type)],
+    label: 'static',
+    color: theme.extend.colors.cyberpunk,
```

Comment:
This is not declarative - when creating a new obstacle, we hard-code that it is static and cyberpunk color.

I think the correct solution would be to not update the list of obstacles [here](https://github.com/viamrobotics/prime/blob/main/packages/blocks/src/lib/navigation-map/components/scene.svelte#L51-L54) and propagate a `add-obstacle` event. Then we could let the obstacles be uncontrolled and purely declarative.